apps/prairielearn/elements/pl-number-input/pl-number-input.mustache

```diff
@@ -86,6 +108,13 @@
 {{#label}}<span>{{{label}}}</span>{{/label}}
 <samp>{{a_sub}}</samp>
 {{#suffix}}<span>{{suffix}}</span>{{/suffix}}
+{{#feedback}}
+<div class="d-inline-flex pl-number-input-feedback">
+  {{feedback}}
+</div>
+{{/feedback}}
+
+
 {{#correct}}
 <script>
     $(function(){
```

Comment:
I don't think the stuff in this script tag is needed anymore?  The correct badge doesn't need to be a popover anymore. This should be just a badge, the way that the partial and incorrect tags below are set up.

Reply:
That makes sense. Fixed in [82622b3](https://github.com/PrairieLearn/PrairieLearn/pull/7934/commits/82622b3b92a4429aa9c83c500d60c47d77c9b474).